src/Files.App/Data/Items/ListedItem.cs

```diff
@@ -9,6 +9,7 @@
 using System.Drawing;
 using System.IO;
 using System.Text;
+using Vanara.PInvoke;
```

Comment:
Can you please remove this reference?

Reply:
It is currently required for the enum ShowWindowCommand. I could create a separate enum that contains the same values. Then the enum from Vanara.PInvoke would only be used when reading/updating the files and we cast from/to our enum. Or what do you think?

Reply:
We're currently working to remove `Vanara`.

Reply:
Replaced it with SHOW_WINDOW_CMD from Windows.Win32.UI.WindowsAndMessaging